int64 id_KeyFrame
msg_feature[] features_left
msg_feature[] features_right
geometry_msgs/PoseWithCovariance cameraPose
bool bFixed